multimodal/src/autogluon/multimodal/configs/model/fusion_mlp_image_text_tabular.yaml

```diff
@@ -149,6 +149,7 @@ model:
       - "image"
     max_img_num_per_col: 1
     output_bbox_format: "xyxy"  # now support xyxy or xywh, for bbox format details see https://keras.io/api/keras_cv/bounding_box/formats/
+    frozen_layers: null
```

Comment:
What if removing `null`? What's the difference between nothing and `null`?

Reply:
I think there's no difference. We are using both in the configs.